### PR TITLE
RUN-14761-ConfigMap-data-source-controlled-by-policies

### PR DESCRIPTION
**Pull request by jasonnovichRunAI**

### CODE REVIEW

File: docs/snippets/snippets-policies.md [hunk 1]
```diff
@@ -168,6 +169,13 @@ The `defaults` section of the policy file is...
 | `accessKeyAssetId` | `string`<uuid> or `null` non-empty | ID of credentials asset of type access-key, for private S3 buckets. |
 | `url` | `string` or `null` non-empty | The url of the S3 service provider. The default is the URL of the Amazon AWS S3 service. |
 
+#### ConfigMap Resource Description Fields
+
+|Field | Type | Description |
+| -- | -- | --|
+|`json:"configMap"` | `string` | The name of the ConfigMap resource. ConfigMap is mandatory for creating a workspace. |
```
Comment (shmuel-runai) [inline docs/snippets/snippets-policies.md:176]: `The name of the ConfigMap`

File: docs/snippets/snippets-policies.md [hunk 2]
```diff
@@ -168,6 +169,13 @@ The `defaults` section of the policy file is...
...
+|`json:"mountPath"` | `string` | Local path within the workspace to which the PVC bucket will be mapped. ClaimName is mandatory for creating a workspace. |
```
Comment (shmuel-runai) [inline docs/snippets/snippets-policies.md:177]: `Local path within the workspace to which the ConfigMap will be mapped`

**Review by shmuel-runai — COMMENTED** [review]
minor fix

File: docs/snippets/snippets-policies.md (hunk 2 above)
Comment (jasonnovichRunAI) [reply docs/snippets/snippets-policies.md:177]:
```suggestion
|`json:"mountPath"` | `string` | Local path within the workspace to which the ConfigMap bucket will be mapped. ClaimName is mandatory for creating a workspace. |
```

File: docs/snippets/snippets-policies.md [hunk 3]
```diff
@@ -168,6 +169,13 @@ The `defaults` section of the policy file is...
 | `accessKeyAssetId` | `string`<uuid> or `null` non-empty | ID of credentials asset of type access-key, for private S3 buckets. |
 | `url` | `string` or `null` non-empty | The url of the S3 service provider. The default is the URL of the Amazon AWS S3 service. |
 
+#### ConfigMap Resource Description Fields
+
+|Field | Type | Description |
+| -- | -- | --|
+|`json:"configMap"` | `string` | The name of the ConfigMap resource. ConfigMap is mandatory for creating a workspace. |
+|`json:"mountPath"` | `string` | Local path within the workspace to which the ConfigMap bucket will be mapped. ClaimName is mandatory for creating a workspace. |
```
Comment (jasonnovichRunAI) [inline docs/snippets/snippets-policies.md:177]:
```suggestion
|`json:"mountPath"` | `string` | Local path within the workspace to which the ConfigMap will be mapped. ClaimName is mandatory for creating a workspace. |
```

File: docs/snippets/snippets-policies.md (hunk 1 above)
Comment (jasonnovichRunAI) [reply docs/snippets/snippets-policies.md:176]:
```suggestion
|`json:"configMap"` | `string` | The name of the ConfigMap. ConfigMap is mandatory for creating a workspace. |
```

**Pull request merged**